nvidia_deeplearningexamples_tacotron2.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: NCF
+summary: The Tacotron 2 model for generating mel spectrograms from text
+category: researchers
+image: nvidia_logo.png
+author: NVIDIA
+tags: [audio]
+github-link: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/Tacotron2
+featured_image_1: no-image
+featured_image_2: no-image
+---
+
+```python
+import torch
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_tacotron2')
```

Comment:
These two args are positional args, you can remove the keywords I think

Reply:
ok